### ISSUE TITLE
On/Off display?

### ISSUE THREAD

**Issue by anar4732** [Electrical-Age/ElectricalAge#488]
http://i.hizliresim.com/nrdjMM.png

**Comment by cm0x4D** [comment]
Makes sense for me with a configurable text for each state which defaults to On, respective Off...

**Comment by Baughn** [comment]
If we're making feature requests, then it'd be great if it could alternately render the current voltage/power/whatever of a signal, similarly to the data loggers. ;-)
